### ISSUE TITLE
Xcode 7 bug

### ISSUE THREAD

**Issue by emzielonka** [robbiehanson/XcodeColors#58]
I'm having problems with the colors in Xcode 7.0 - console prints all in white again.

**Comment by Reggian** [comment]
Same with 7.0.1.
But it works just fine with 7.1 (beta) so this might be an Xcode 7.0.x bug or something.

**Comment by zhuli8com** [comment]
I'm having problems with the colors in Xcode 7.0 too!

**Comment by vincentsong** [comment]
it works in 7.0 but not in 7.1

**Comment by vincentsong** [comment]
reinstall Alcatraz and install this will work

**Comment by emzielonka** [comment]
Well I don't use Alcatraz.

**Comment by dslysenko** [comment]
Uninstalling XcodeColors in Alcatraz, reinstalling it and reopening Xcode fixed it for me.

**Comment by emzielonka** [comment]
Fixed it! (with no Alcatraz-thing)
The console printed:
"[MT] PluginLoading: Required plug-in compatibility UUID 7265231C-39B4-402C-89E1-16167C4CC990 for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/XcodeColors.xcplugin' not present in DVTPlugInCompatibilityUUIDs"
so I added the the given UUID to the .plist file in XcodeColors project. After Xcode restart, everything's beautifully colourful back again.
